### DISCUSSION TITLE
SignServerException calling signer with ID 2 : Error constructing PKCS7 package

### DISCUSSION THREAD

**Discussion by nguyenthanhvu240** [Keyfactor/signserver-ce#58] [General]
I have problems signing pdf files. I received this notification "SignServerException calling signer with ID 2 : Error constructing PKCS7 package ".Signer with ID 2 is PDFSigner (2) and it had active.
I tested pdf file signing and it was successful based on  "https://hub.docker.com/r/keyfactor/signserver-ce".
Then I did another way based on this video. I didn't make PlainSigner, instead I made PDFSigner "https://www.youtube.com/watch?v=sNIv0nYRs1g"
I don't use pkcs7,i commented pkcs12,i use Internal key.
![image](https://github.com/Keyfactor/signserver-ce/assets/49347804/41e72248-ebd1-4852-b79b-2f3329906dde)
![image](https://github.com/Keyfactor/signserver-ce/assets/49347804/b9cd0320-7933-4ca1-aaf2-ebcdad4a0a83)
![image](https://github.com/Keyfactor/signserver-ce/assets/49347804/9a8f44a6-6496-4de2-b902-1e4d6b42f0a5)

**Comment by netmackan** [comment]
For HTTP 500 / Internal Server Error / SignServerException you need to check the server log for more details. Please check the console output from the container for the stacktrace and especially look for the "caused by" part(s) which could give more details about the problem. You can also paste the full stacktrace here.
